modules/logging-release-notes-5-8-2.adoc

```diff
@@ -0,0 +1,26 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-2"]
+= Logging 5.8.2
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0271[OpenShift Logging Bug Fix Release 5.8.2].
+
+[id="logging-release-notes-5-8-2-bug-fixes"]
+== Bug fixes
+Before this update, the LokiStack ruler pods would not format the IPv6 pod IP in HTTP URLs used for cross pod communication, causing querying rules and alerts through the Prometheus-compatible API to fail. With this update, the LokiStack ruler pods encapsulate the IPv6 pod IP in square brackets, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4890[LOG-4890])
+
+* Before this update, the developer console's logs did not account for the current namespace, resulting in query rejection for users without cluster-wide log access. With this update, namespace inclusion has been corrected, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4947[LOG-4947])
+
+* Before this update, the logging view plugin of the {Product-Title} web console did not allow for custom node placement and tolerations. With this update, defining custom node placements and tolerations has been added to the logging view plugin of the {Product-Title} web console. (link:https://issues.redhat.com/browse/LOG-4912[LOG-4912])
+
+* Before this update, in {Product-Title} 4.15, the Loki ruler was not able to send alerts to  `Alertmanager` user workload monitoring due to a permissions issue. With this update, the Loki Operator RBAC permissions allow sending alerts to `Alertmanager`, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4951[LOG-4951])
```

Comment:
I'm wondering about "before OCP 4.15" here. When is this PR supposed to be merged? OCP 4.15 is still a little over a month away, so if mentioning it now is premature.

Reply:
It should have already merged, good catch! I'll add a 'release candidate' distinction, since those builds are available, presumably for customer testing, so this bug has value in that context.

Reply:
I'm not certain that qualifying with Release Candidate is okay...the RH Supplementary SG has clear guidance not to mention future releases: https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future. I'm going to raise this with the merge review squad and see if anyone else has different takes or experience.